community/bolt/src/main/java/org/neo4j/bolt/v2/messaging/Neo4jPackV2.java

```diff
@@ -136,10 +137,10 @@ public void writeLocalTime( long nanoOfDay ) throws IOException
         }
 
         @Override
-        public void writeTime( long nanosOfDayLocal, int offsetSeconds ) throws IOException
+        public void writeTime( long nanosOfDayUTC, int offsetSeconds ) throws IOException
         {
             packStructHeader( 2, TIME );
-            pack( nanosOfDayLocal );
+            pack( TimeConstants.asValidLocalTime( nanosOfDayUTC ) );
```

Comment:
I believe this will change the serialized format to be the UTC local time, correct?
Is that really what we want for the BOLT format? I have a feeling that it might be easier for clients if we sent the localtime within the particular offset.

Reply:
Yes, and when I changed it from UTC to local the first time I broke it too. Was trying to revert that brake, and have communicated with the driver team about it. 

In Drivers they're not sure what it the best format to get, as temporal support in a the driver languages is very different. I think they need to find out what works best, and then change this to what they really want. In short, they're on top on it.

But to cause the least chaos, I changed this PR to _not_ change the serialization. It's probably easier if they do everything from now, and I keep my hand off :).